src/main/thrift/tempest.thrift

```diff
@@ -27,20 +39,20 @@ struct BidirectionalPPRParams {
 }
 
 service TempestService {
-  int outDegree(1:int id)
-  int inDegree(1:int id)
+  int outDegree(1:int id) throws (1:InvalidNodeIdException e)
+  int inDegree(1:int id) throws (1:InvalidNodeIdException e)
 
-  list<int> outNeighbors(1:int id)
-  list<int> inNeighbors(1:int id)
+  list<int> outNeighbors(1:int id) throws (1:InvalidNodeIdException e)
+  list<int> inNeighbors(1:int id) throws (1:InvalidNodeIdException e)
 
   /* Returns the ith out-neighbor of the given node.
      Throws an exception unless 0 <= i < outDegree(id).
    */
-  int outNeighbor(1:int id, 2:int i)
+  int outNeighbor(1:int id, 2:int i) throws (1:InvalidNodeIdException ex1, 2:InvalidIndexException ex2)
```

Comment:
Are these changes also going in the tempest public repo? I am assuming not, for now, but that they will ultimately. Can you please make that an issue in the public repo?